test/integration/full/controls/gcloud.rb

```diff
@@ -12,7 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ENV['CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE'] = attribute('credentials_file_path')
+#ENV['CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE'] = attribute('credentials_file_path')
+
+credentials_file_path = attribute('credentials_file_path')
```

Comment:
```suggestion
```

Reply:
I've removed all part related to this variable and checked the work, full cycle passed, however I would like to listen to your opinion.